packages/cipherstash-proxy/src/config/tandem.rs

```diff
@@ -176,12 +176,14 @@ impl TandemConfig {
         // For parsing top-level values such as CS_HOST, CS_PORT
         // and for parsing nested env values such as CS_DATABASE__HOST, CS_DATABASE__PORT
         let cs_env_source = Environment::with_prefix(CS_PREFIX)
+            .ignore_empty(true)
```

Comment:
docker-compose will pass an empty value if an `ENV` var is not set.
Proxy will now ignore any empty values.